revscores/language/turkish.py

```diff
@@ -0,0 +1,76 @@
+import warnings
+
+from nltk.corpus import wordnet
+from nltk.stem.snowball import SnowballStemmer
+
+from .language import Language
+
+STEMMER = SnowballStemmer("turkish")
+
+BADWORDS = set(STEMMER.stem(w) for w in [
+    "abracao",
+    "Ağzına sıçayım",
+    "Ahlaksız",
+    "Ahmak",
+    "Am",
+    "Amcık",
+    "Amın oğlu",
+    "Amına koyayım",
+    "Amına koyyim",
+    "Amk",
+    "Aptal",
+    "Beyinsiz",
+    "Bok",
+    "Boktan",
+    "Çük",
+    "Dedeler",
+    "Embesil",
+    "Gerizekalı",
+    "Gerzek",
+    "Göt",
+    "Göt oğlanı",
+    "Götlek",
+    "Götoğlanı",
+    "Götveren",
+    "Haysiyetsiz",
+    "İbne",
+    "İnci",
+    "İt oğlu it",
+    "Kıç",
+    "Mal",
+    "Meme",
+    "Nobrain",
+    "Oğlan",
+    "Oğlancı",
+    "Orospu",
+    "Orospu çocuğu",
+    "Orospunun evladı",
+    "Pezevengin evladı",
+    "Pezevenk",
+    "Piç",
+    "Puşt",
+    "Salak",
+    "Şerefsiz",
+    "Sik",
+    "Siktir",
+    "Yarrak"
+])
+
+class Turkish(Language):
```

Comment:
It seems you are using an outdated version of the repo as basis for writing this. The format for the language files changed some time ago. Take a look in the [current English file](https://github.com/halfak/Revision-Scoring/blob/9d46c556d724fb4769f2059136ad9c4979caa79c/revscoring/languages/english.py#L105-L119).